enzy_htp/core/math_helper.py

```diff
@@ -0,0 +1,85 @@
+"""Module holds functions for math calculations. 
```

Comment:
I won't annotate every problem with every functions in this file, but please add the type hinting as well as standardized docstrings.